gravatar/src/test/java/com/gravatar/GravatarSdkContainerRule.kt

```diff
@@ -24,7 +24,6 @@ class GravatarSdkContainerRule : TestRule {
                 gravatarApiServiceMock = mockk<GravatarApiService>(relaxed = true)
                 mockkObject(GravatarSdkContainer)
                 every { GravatarSdkContainer.instance } returns gravatarSdkContainerMock
-                every { gravatarSdkContainerMock.dispatcherDefault } returns testDispatcher
```

Comment:
Nice! It's probably worth also mocking the other dispatchers to also use the `testDispatcher`, which is usually what you want in most tests.